src/sys/resource.rs

```diff
@@ -293,7 +293,9 @@ impl Usage {
         TimeVal::from(self.0.ru_stime)
     }
 
-    /// The resident set size at its peak, in kilobytes.
+    /// /// The resident set size at its peak,
```

Comment:
Seems to be redundant:

```suggestion
    /// The resident set size at its peak,
```